index.js

```diff
@@ -38,8 +38,51 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const wrapArg = item => {
+	const alfredworkflow = {arg: item.arg, variables: item.variables};
+	const arg = JSON.stringify({alfredworkflow});
+	const newItem = Object.assign({}, item, {arg});
+	delete newItem.variables;
+	return newItem;
+};
+
+const formatMods = item => {
+	const copy = Object.assign({}, item);
+
+	Object.keys(item.mods).forEach(modKey => {
+		const modItem = item.mods[modKey];
+		copy.mods[modKey] = wrapArg(modItem);
+	});
+
+	return copy;
+};
+
+// see https://www.alfredforum.com/topic/9070-how-to-workflowenvironment-variables/
+// for documentation on setting environment variables from Alfred workflows
+const format = item => {
+	if (!isDefined(item)) {
+		return item;
+	}
+
+	if (isDefined(item.variables)) {
+		item = wrapArg(item);
+	}
+
+	if (isDefined(item.mods)) {
+		item = formatMods(item);
+	}
+
+	return item;
+};
+
+alfy.output = items => {
+	if (Array.isArray(items)) {
```

Comment:
To come back to the `throw new TypeError()` thing. Just wondering, is there a reason why you would use `alfy.output` without providing an `Array`? If not, I think I like to be more explicit about it and throw an error instead.

// @sindresorhus